### PR TITLE
Add game development stub

### PR DESCRIPTION
**Pull request by HKuz**
Add a stub in the Game Development folder and commits latest changes from the pre-commit normalise script 😄

### CODE REVIEW

File: src/pages/machine-learning/linear-regression/index.md [hunk 1]
```diff
@@ -7,39 +7,9 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
-### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
-
-Linear regression helps us predict score of a variable X from the scores on other variables Y. When the variables Y are plotted, linear regression finds the best-fitting straight line through the points. The best-fitting line is called a regression line.
-
-[Online linear regression simulator](https://www.mladdict.com/linear-regression-simulator)
+<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
```
Comment (systimotic) [inline src/pages/machine-learning/linear-regression/index.md]: This deletes the entire article 😅

File: src/pages/visual-design/visual-hierarchy/index.md [hunk 2]
```diff
@@ -12,4 +12,4 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
 
-<a href='https://www.interaction-design.org/literature/article/visual-hierarchy-organizing-content-to-follow-natural-eye-movement-patterns' target='_blank' rel='nofollow'>www.interaction-design.org: Visual Hierarchy: Organizing content to follow natural eye movement patterns</a>
+
```
Comment (systimotic) [inline src/pages/visual-design/visual-hierarchy/index.md]: Same as the others

File: src/pages/computer-science/what-is-an-sdk/index.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
-The term "SDK" stands for Sofware Development Kit.
+
```
Comment (systimotic) [inline src/pages/computer-science/what-is-an-sdk/index.md]: This removes the content added to this Guide. In this case, I'm okay with that.

File: src/pages/computer-science/what-is-an-api/index.md [hunk 4]
```diff
@@ -9,9 +9,7 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
-The term "API" stands for Application Programming Interface.
-
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
 
-[https://medium.freecodecamp.org/what-is-an-api-in-english-please-b880a3214a82](https://medium.freecodecamp.org/what-is-an-api-in-english-please-b880a3214a82)
+
```
Comment (systimotic) [inline src/pages/computer-science/what-is-an-api/index.md]: The content added here is also removed.

File: src/pages/computer-science/what-is-an-sdk/index.md (hunk 3 above)
Comment (HKuz) [reply src/pages/computer-science/what-is-an-sdk/index.md]: 👍 That's fair, it's not adding much

File: src/pages/machine-learning/linear-regression/index.md (hunk 1 above)
Comment (HKuz) [reply src/pages/machine-learning/linear-regression/index.md]: Yikes! 😱

**Comment by HKuz** [comment]
No problem! Back to QA'ing some PRs...

**Pull request merged**